includes/apple-exporter/components/class-podcast.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Publish to Apple News Includes: Apple_Exporter\Components\Podcast class
+ *
+ * Contains a class which is used to transform Podcast embeds into Apple News format.
+ * Documentation about apple podcast players can be found here: https://podcasters.apple.com/support/889-apple-podcasts-embed-player.
+ *
+ * @package Apple_News
+ * @subpackage Apple_Exporter
+ * @since 0.2.0
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * A class to transform a Podcast embed into an Apple News Podcast component.
+ *
+ * @since 0.2.0
+ */
+class Podcast extends Component {
+
+	/**
+	 * Look for node matches for this component.
+	 *
+	 * @param \DOMElement $node The node to examine for matches.
+	 * @access public
```

Comment:
The `@access` phpdoc is a relic and should be removed from new code. Apply globally to files in this PR.